docs/development.rst

```diff
@@ -539,6 +539,99 @@ For further discussion of good and bad practices regarding commits, see:
 .. _How to Get Your Change Into the Linux Kernel: https://www.kernel.org/doc/Documentation/SubmittingPatches
 
 
+.. _`Making a change`:
+
+Making a change
+---------------
+
```

Comment:
Add the following:   All changes to pywbem are made through github with PRs created on topic branches and merged with the current master after successful group review.

Reply:
DONE as proposed.